pos_deposit/models/product_template.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Sunflower IT
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    use_deposit = fields.Boolean("Use Deposit")
```

Comment:
I don't see the interest of this field. If the deposit product is defined, it is using deposit.